channels/rdpsnd/server/rdpsnd_main.c

```diff
@@ -591,24 +595,18 @@ void rdpsnd_server_context_reset(RdpsndServerContext *context)
 
 void rdpsnd_server_context_free(RdpsndServerContext* context)
 {
-	if (!context->priv->StopEvent)
```

Comment:
@hardening looks like I overlooked that one. Should never be NULL, but the remove was accidental.